app/views/customers/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div style="text-align: center;border-bottom: 2px solid;padding-top: 10px;padding-bottom:9px;margin-bottom: 10px;"> listing all order 
+</div>
```

Comment:
@amaninquirly. Inline CSS is bad practice. use controller specific css files. (i.e) customer.css and then include those in  application layout.